plugins/modules/public_cloud_failover_ip_info.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: public_cloud_failover_ip_info
+short_description: Retrieve all info for a OVH failover IP
+description:
+    - This module retrieves all info from a OVH failover IP
+author: Article714 (M. Piriou, C. Guychard)
+requirements:
+    - ovh >= 0.5.0
+options:
+    service_name:
+        required: true
+        description: The OVH API service_name is  Public cloud project Id
+    fo_ip:
+        required: true
+        description:
+            - The fail-over IP
+
+'''
+
+EXAMPLES = '''
+synthesio.ovh.public_cloud_failover_ip_info:
+  fo_ip: "{{ fo_ip }}"
+  service_name: "{{ service_name }}"
+delegate_to: localhost
+register: fo_ip_id
```

Comment:
The register here is a little bit confusing for me: the module returns the whole failover informations, and you are using the `fo_ip_id` in other modules, where it must be the instance id

Reply:
It's an example.